src/Atmos/Model/AtmosModel.jl

```diff
@@ -619,8 +619,14 @@ function transform_post_gradient_laplacian!(
     )
 end
 
-precompute(atmos::AtmosModel, args, ::Flux{SecondOrder}) =
-    (ts = recover_thermo_state(atmos, args.state, args.aux),)
+function precompute(atmos::AtmosModel, args, ::Flux{SecondOrder})
+    @unpack state, diffusive, aux, t = args
+    ν, D_t, τ = turbulence_tensors(atmos, state, diffusive, aux, t)
+    return (
+        ts = recover_thermo_state(atmos, state, aux),
+        turbulence = (ν = ν, D_t = D_t, τ = τ),
+    )
+end
```

Comment:
Is there any experiment out there that _doesn't_ use these for second order flux computation?

Reply:
The `AtmosModel` hard-codes using `ViscousStress{Momentum}()` and `ViscousStress{Energy}()` (which was the case before the tendency specification layer was added). So `turbulence_tensors` is always needed/called. If/when we want to make this more flexible, we can change this to `turbulence = precompute(atmos.turbulence, atmos, args, Flux{SecondOrder})` and compute this, or not, depending on the turbulence model.

Reply:
It looks like we can remove `ts` here for all cases except when the EDMF model is used. So we could use `ts = atmos.turbconv isa EDMF ? recover_thermo_state(atmos, state, aux) : NamedTuple()`.

Reply:
😢 If we must.

Reply:
I don't think we need to. I'm fine with keeping it as is